docs/index.md

```diff
@@ -20,6 +20,7 @@
 * `exclude_files`: A list of files containing exclude lists
 * `tags`: A list of tags for the new snapshot
 * `dry_run`: Set to `True` to perform just a dry run of the backup
+* `host`: Set the hostname for the snapshot manually. To prevent an expensive rescan use the "parent" flag
```

Comment:
Can we remove the "To prevent an expensive rescan..." part since we don't yet support the `parent` flag?